gh-actions/go/generate/action.yaml

```diff
@@ -29,6 +29,10 @@ runs:
       env:
         DEBIAN_FRONTEND: noninteractive
       run: |
+        if command -v jq &> /dev/null; then
```

Comment:
The steps will be named "if command…".

I suggest changing the "Download jq" echo to "Ensure jq is installed", and put that at the first line.

Also, then, your commands will be after `set -eu`